core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp

```diff
@@ -48,6 +48,30 @@
             "roleType": "NIFI_NODE",
             "base": true,
             "configs": [
+              {
+                "name": "ssl_enabled",
+                "value": "true"
+              },
+              {
+                "name": "ssl_server_keystore_location",
```

Comment:
this will only helps us in case of default templates

Reply:
@doktoric yep, agree, i think that information could be added to documentation and also we should double check CM will resolve related jira to make that workable for non default cases.
@pvillard31 what do you think about that?

Reply:
I think we need an ETA on the CM fix because we dont want to add such thing in the future.

Reply:
@doktoric agree that it should be fixed by CM for future releases but anyway ssl/tls properties that were added, should not break anything, correct?

Reply:
Do we have a proper JIRA to track this with the CM team? I think Matt S. said that it would be fixed in the next release of CM but we need confirmation.

What does it mean that it'll only work "in case of default templates"? I would not expect anyone to create a template with an empty group for scaling, no? What should be documented exactly?

Reply:
@pvillard31 as i understand it should be fixed here https://jira.cloudera.com/browse/OPSAPS-54928, but there is no fix version and any ETA in this jira. About "in case of default templates", as i understand some customers using ability to create their own templates for datahub clusters, so scale up could fail for them, because they don't know about related ssl/tls properties and new hostGroup with cardinality 0 should be added. But if we expect users will not create custom templates, then nothing to document. @doktoric what do you think? Could this patch be merged?

Reply:
OPSAPS-59770 is the ticket what I raised with blocker.
@vbrodetskyi I can merge but honestly I would like to remove this as soon as possible. So I would like to see an ETA on the CM ticket. If we dont get a fix for it then we need to add this configuration to the next versions as well.

Reply:
@doktoric agree, i will remove this fix from 7.2.9 and 7.2.10. I've added comment about related OPSAPS jira https://jira.cloudera.com/browse/CFM-1739?focusedCommentId=4351063&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4351063.

Reply:
@doktoric i've updated patch and all checks have passed. If you agree, could you please merge changes? Also could you please clarify when they will be available on mow-dev? Thank you.